lib/iaas_web/templates/page/index.html.eex

```diff
@@ -1,21 +1,10 @@
 <section class="phx-hero">
-  <h1>Identicons as a Service</h1>
-  <div class="center">
-    <img id="identicon" src="<%= Routes.page_path(@conn, :image, @text, @size) %>" alt="A <%= @size %>x<%= @size %> identicon from '<%= @text %>'" title="<%= @text %>" class="card mw-300" />
-    <figcaption id="caption">A <%= @size %>x<%= @size %> identicon from <strong><%= @text %></strong></figcaption>
-  </div>
-  <div>
-    <label for="text">Text</label>
-    <input id="text" type="text" value="<%= @text %>">
-    <label for="size">Size</label>
-    <select id="size">
-      <%= for number <- 1..10 do %>
-        <%= if number == @size do %>
-          <option value="<%= number %>" selected="selected"><%= number %></option>
-        <% else %>
-          <option value="<%= number %>"><%= number %></option>
-        <% end %>
-      <% end %>
-    </select>
-  </div>
+  <div id="app"></div>
+  <script>
+    const text = `<%= @text %>`;
+    const size = <%= @size %>;
+    
+    window[Symbol.for('text')] = text;
+    window[Symbol.for('size')] = size;
```

Comment:
Rather than storing the `text` and `size` in a global variable, can we assign the values directly to the symbols directly? Honestly, it would be nice if we could initialize the Vue app with with those values... I fiddled with it a little bit without much luck.
```vue
<div id="app" :text="<%= @text %>" :size="<%= @size %>"></div>
```


```suggestion
    window[Symbol.for('text')] = '<%= @text %>';
    window[Symbol.for('size')] = <%= @size %>;
```

Reply:
I think that a better way to do this would be to use a `<script type="module">` here that would enable us to import the `Vue` app and the `MainLayout` component. This would enable keeping `window` untouched.

Reply:
This absolutely does not work, I don't know what I was thinking.